providers/cloudflare/cloudflareProvider.go

```diff
@@ -227,6 +234,11 @@ func (c *cloudflareProvider) GetDomainCorrections(dc *models.DomainConfig) ([]*m
 				Msg: d.String(),
 				F:   func() error { return c.deletePageRule(ex.Original.(*pageRule).ID, id) },
 			})
+		} else if ex.Type == "WORKER_ROUTE" {
```

Comment:
Let's consistently use `CF_WORKER_ROUTE` and not `WORKER_ROUTE` (or are they 2 different things?)

Reply:
2 different things. See last comment.